corehq/apps/app_manager/models.py

```diff
@@ -3507,6 +3498,21 @@ def get_filter_value(self, user, ui_filter):
         return None
 
 
+class AncestorLocationTypeFilter(ReportAppFilter):
+    ancestor_location_type_name = StringProperty()
```

Comment:
It would be *really* nice to set `choices` for the domain. I can't see a simple way to do that though.